vulnfeeds/cmd/alpine/main.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/google/osv/vulnfeeds/vulns"
+)
+
+const (
+	alpineURLBase    = "https://secdb.alpinelinux.org/%s/main.json"
+	alpineIndexURL   = "https://secdb.alpinelinux.org/"
+	alpineOutputPath = "parts/alpine_output"
+)
+
+func main() {
+	flag.Parse()
+
+	err := os.MkdirAll(alpineOutputPath, 0755)
+	if err != nil {
+		log.Fatalf("Can't create output path: %s", err)
+	}
+
+	generateAlpineOSV()
+}
+
+func getAllAlpineVersions() []string {
```

Comment:
Could you please add a descriptive comment per function?

See also https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences